easybuild/easyconfigs/b/bzip2/bzip2-1.0.6-GCC-5.2.0.eb

```diff
@@ -0,0 +1,19 @@
+name = 'bzip2'
+version = '1.0.6'
+
+homepage = 'http://www.bzip.org/'
+description = """bzip2 is a freely available, patent free, high-quality data compressor. It typically 
+compresses files to within 10% to 15% of the best available techniques (the PPM family of statistical 
+compressors), whilst being around twice as fast at compression and six times faster at decompression."""
+
+toolchain = {'name': 'GCC', 'version': '5.2.0'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.bzip.org/%(version)s/']
+
+import os
+os.environ['CC'] = 'gcc'
+os.environ['CFLAGS'] = '-O3 -fPIC'
```

Comment:
I've added that bzip2 config as EB was complaining that it couldn't resolve the bzip2 dependency when I ran `eb Boost-1.59.0-GCC-5.2.0.eb -r`. Thus I've just copied `bzip2-1.0.6-GCC-4.9.2.eb` and adjusted GCC to 5.2.0.